extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -238,6 +238,11 @@
 }
 
 /** For amp-story-outlink-page-attachment-v2 experiment elements. */
+.i-amphtml-story-page-open-attachment {
+  --i-amphtml-outlink-cta-background-color: white !important;
+  --i-amphtml-outlink-cta-text-color: black !important;
+}
```

Comment:
There's another place in this file with this selector, we should move these over there (it even has the same comment)

Reply:
Actually it's there 3 times but each time, the vars are for one specific UI. The comments aren't the exact same, the one above is for INLINE-page-attachments and this is OUTLINK. I think leaving them separate and grouped with the styles they're used by is most useful since this style sheer is for all the UIs.